js/index.html

```diff
@@ -37,7 +37,7 @@ <h2>Message Type</h2>
             <!-- <label>Supported versions: <input type="text" name="versions" id="" value="0xc0000000ff000001"></label> -->
             <label>Supported versions: <input type="text" name="versions" id="" value="0xff000001" /></label>
             <br />
-            <label>Track NameSpace: <input type="text" name="track-namespace" id="" value="vc" /> </label>
+            <label>Track NameSpace: <input type="text" name="track-namespace" id="" value="vc/aa" /> </label>
```

Comment:
Is this a remnant of the testing?

Reply:
`vc/aa` is example. 
I'll replace `vc/aa` to `vc_track_namespace/hoge`.

Reply:
> vc/aa 

+1 for the reason that it is understandable as a sample app behavior